StyleCop.Analyzers/StyleCop.Analyzers.Test/NamingRules/SA1300UnitTests.cs

```diff
@@ -44,6 +44,31 @@ public async Task TestLowerCaseNamespaceAsync()
             await VerifyCSharpFixAsync(testCode, expected, fixedCode, CancellationToken.None).ConfigureAwait(false);
         }
 
+        [Fact]
+        public async Task TestAllowedLowerCaseNamespaceIsNotReportedAsync()
+        {
+            var customTestSettings = @"
+{
+  ""settings"": {
+    ""namingRules"": {
+      ""allowedNamespaceComponentTerms"": [ ""eBay"" ]
```

Comment:
❓ Would it be better for this to be `allowedNamespaces`? For the `iPod` example below, the configuration would be `Apple.iPod`, which would then allow `Apple.iPod` itself along with any namespace under it, but would not allow other top-level namespaces like `Android.iPod`.

Reply:
I was hoping to configure namespace components, so that `Apple.iPod` and `Android.iPod` would both be acceptable.